app/src/androidTest/java/org/simple/clinic/benchmark/bp/BloodPressureEntryBenchmark.kt

```diff
@@ -26,20 +28,29 @@ class BloodPressureEntryBenchmark : BaseBenchmarkTest() {
   @Inject
   lateinit var uuidGenerator: UuidGenerator
 
+  @Inject
+  lateinit var patient: Patient
```

Comment:
We cannot inject a patient here 😅. We can find a patient ID from the facility, either from the dashboard or signing in into our test account and getting it manually.